VirtoCommerce.CatalogModule.Web/Scripts/blades/property-dictionary-details.js

```diff
@@ -0,0 +1,154 @@
+angular.module('virtoCommerce.catalogModule')
+    .controller('virtoCommerce.catalogModule.propertyDictionaryDetailsController',
+        ['$scope', 'platformWebApp.dialogService', 'platformWebApp.bladeNavigationService', 'platformWebApp.common.languages',
+            function ($scope, dialogService, bladeNavigationService, languages) {
+                var blade = $scope.blade;
+                var pb = $scope.blade.parentBlade;
+                var languagesList = languages.query();
```

Comment:
We must pass all that we need in this blade via blade properties (input parameters) and avoid of access to them through parent blade directly.